Adyen/UI/Form/Items/Address/AddressViewModel.swift

```diff
@@ -55,6 +55,17 @@ public struct AddressViewModel {
 
 }
 
+@_spi(AdyenInternal)
+public extension AddressViewModel {
+    
+    /// Returns all fields that are not specified as `optionalFields`
+    var requiredFields: Set<AddressField> {
```

Comment:
Can we test these changes for countries like "US" , "CA", "GB" and "BR" because they have different address forms?